tl-tl/sc/introduction.md

```diff
@@ -1,34 +1,34 @@
-# NEO Smart Contract Introduction
+# Panimula sa Neo Smart na Kontrata
 
-## What is a smart contract?
+## Ano ang isang smart na kontrata?
 
-A smart contract is a set of commitments that are defined in digital form, including the agreement on how contract participants shall fulfill these commitments. Blockchain technology gives us a decentralized, non-tampering, highly reliable system in which smart contracts are extremely useful. Smart contracts is one of the most important characteristics of blockchain technologies and the reason why blockchains can be called disruptive technology. It is increasing the efficiency of our social structure by each passing day.
+Ang isang smart na kontrata ay isang pangkat ng mga pangako na itinakda sa digital na anyo, kabilang nito ang kasunduan kung paano ang mga kasali sa kontrata ay tutupad sa mga pangakong ito. Ang blockchain na teknolohiya ay nagbibigay sa atin ng isang desentralisado, `di pakikialamang, labis na maaasahang sistema kung saan ang mga smart na kontrata ay lubhang kapakipakinabang. Ang mga smart na kontrata ay isa sa mga pinakaimportanteng katangian ng mga blockchain na teknolohiya at ang dahilan kung bakit ang mga blockchain ay maaring matawag na disruptive na teknolohiya. Ito ay nagdadagdag ng efficiency sa ating sosyal na istraktura sa bawat pagdaan ng araw.
 
-## What are the characteristics of NEO smart contracts?
+## Ano ang mga katangian ng NEO smart na kontrata?
 
-The NEO Smart Contract 2.0 includes the following features: certainty, high performance, and expandability. The contract types include: validation contracts, function contracts, and application contracts.
+Ang NEO Smart Contract 2.0 ay binubuo ng mga sumusunod na mga feature: katiyakan, mataas na lebel na performance, at expendability. Ang mga uri ng kontrata ay binubuo ng mga sumusunod: mga pagpapapatuny na kontrata, mga punsyon na kontrata, at mga aplikasyon na kontrata.
```

Comment:
- `feature` to `tampok`
- `pagpapapatunay `

Reply:
fix provided